wordpress-mobile/WordPress-Aztec-iOS
# Feature - Adds support for the Mark HTML tag

This PR adds support for the `mark` HTML tag.

It currently supports setting the text color and there are plans to add support for background-color as well.

<kbd><img src="https://user-images.githubusercontent.com/4885740/136046091-abd3e3ed-ee4e-46aa-b65b-17ca9f927932.png" width=200 /></kbd>

**To test:**
- Open the demo app
- Check either the standard demo or the Gutenberg demo
- Scroll to the bottom to find the new tag
- Expect to see the text with the colors

**Question**: In the console, I'm still getting `Message: Tag mark invalid` is there something else that's missing to avoid that message?
